src/Salt/UserBundle/Entity/User.php

```diff
@@ -435,9 +439,17 @@ public function suspendUser() {
      *
      * @return $this
      */
-    public function unsuspendUser() {
-        $this->locked = false;
+    public function activateUser() {
```

Comment:
The admin UI needs to be updated to change the current "Unsuspend" action to have the new options.